clusterloader2/pkg/measurement/util/pvcs.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// PVCStartupStatus represents phase of a pvc group.
+type PVCStartupStatus struct {
+	Pending  int
+	Bound    int
+	Lost     int
+	Expected int
+	Created  int
+}
+
+// String returns string representation for PVCsStartupStatus.
+func (s *PVCStartupStatus) String() string {
+	return fmt.Sprintf("PVCs: %d out of %d created, %d bound, %d pending, %d lost", s.Created, s.Expected, s.Bound, s.Pending, s.Lost)
+}
+
+// ComputePVCStartupStatus computes PVCsStartupStatus for a group of PVCs.
+func ComputePVCStartupStatus(pvcs []*corev1.PersistentVolumeClaim, expected int) PVCStartupStatus {
+	startupStatus := PVCStartupStatus{
+		Expected: expected,
+	}
+	for _, p := range pvcs {
+		startupStatus.Created++
+		if p.Status.Phase == corev1.ClaimPending {
```

Comment:
nit: switch statement?

Reply:
was copied from pods.go - may probably be cleaned up, but that's not directly related to this PR

Reply:
ok, this can be a separate PR then